### PR TITLE
Update RPM and CLI links to 0.161

### PR DESCRIPTION
**Pull request by teamsoo**
Presto 0.161 works well on my HDP 2.5 cluster.

It would be good to update the main repo to the new version.

### CODE REVIEW

**Comment by ArturGajowy** [comment]
Thx for the contribution! This changes the default installed version, so probably should be updated in README as well (see this commit: https://github.com/teamsoo/ambari-presto-service/commit/9c82c986a201ac0ed127edf67cf0f0d5da3a20d6).

For sure @petroav would need to review this :)

**Comment by teamsoo** [comment]
Thanks @ArturGajowy for the suggestion. Please see [this commit](https://github.com/prestodb/ambari-presto-service/pull/25/commits/d081d11866ed73f304d8846f6dabb0a90c6df1f4)

File: setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 setup(
     name='ambari-presto',
-    version='1.1',
+    version='1.2',
```
Comment (ArturGajowy) [inline setup.py]: This should probably go in a separate commit - it's not documentation

**Comment by mattsfuller** [comment]
Would there be anyway to generalize this more? This project hasn't done well at keeping in sync with the latest version of Presto. 

Is there a way to specify "latest" so it always inserts the latest version?
Even if we parameterized the version in the build so there is only one place to change, that would be better.

File: setup.py (hunk 1 above)
Comment (teamsoo) [inline setup.py]: Oh sorry, I was following [this commit](https://github.com/prestodb/ambari-presto-service/commit/b69c2e5b225107aa8aee19b43e09bca880af9147) from @petroav

**Comment by petroav** [comment]
@mattsfuller I thought it was possible to get a latest link out of Maven but I just tried it out and couldn't get it to work. Anyway, even if it were possible the problem with this approach is that we don't know if the newest Presto version will be compatible with the integration code in this repo. For example, deprecating properties can cause problems and in fact I ran into this once before. It's a tradeoff between keeping up to date and ensuring everything works out of the box.

@teamsoo thanks for the PR. I don't think we should bump the release version (as you do in your second commit) since the changes we're making are minor and don't really deserve a release. @ArturGajowy what do you think? Should we make a new release?

**Comment by ArturGajowy** [comment]
@petroav let's make a release - after all, the lack of it is what caused this PR :)

**Comment by petroav** [comment]
@teamsoo can you confirm that you manually tested this package with the latest HDP and that everything works? Since we no longer include Ambari integration in the Teradata Presto distribution we rely on the community to keep this project up to date. After you confirm that everything is working I'll merge this PR and create a new release. Thanks!

**Comment by teamsoo** [comment]
@petroav I've been using it for almost 3 weeks now and haven't seen any problem so far. The integration works fine with Teradata ODBC driver and Presto JDBC driver (connecting through both Tableau and SQL Workbench/J).

I'm using HDP-2.5.0.0 (2.5.0.0-1245).

**Comment by petroav** [comment]
Merged, thanks!
